instrumentation/aws-sdk/aws-sdk-1.11/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/awssdk/v1_11/AwsClientInstrumentation.java

```diff
@@ -54,7 +54,8 @@ public static void addHandler(
       if (!hasAgentHandler) {
         handlers.add(
             new TracingRequestHandler(
-                InstrumentationContext.get(AmazonWebServiceRequest.class, RequestMeta.class)));
+                InstrumentationContext.get(
```

Comment:
Thanks for the help @piotr-sumo - in #2508 I will remove the `RequestMeta` class completely. Do you mind reverting the changes to the `awssdk` in this PR?

Reply:
@anuraaga sure, no problem. I will revert those changes.